modules/instrument_list/php/instrument_list.class.inc

```diff
@@ -101,33 +125,13 @@ class Instrument_List extends \NDB_Menu_Filter
         ServerRequestInterface $request,
         RequestHandlerInterface $handler
     ) : ResponseInterface {
-        $gets = $request->getQueryParams();
-
-        $attribute = $request->getAttribute("CandID");
-        if ($attribute !== null) {
-            $this->candidate = \Candidate::singleton($attribute);
-        } else {
-            $this->candidate = \Candidate::singleton($gets['candID']);
+        $this->candidate = $request->getAttribute("Candidate");
+        $this->timePoint = $request->getAttribute("timePoint");
+        if ($this->candidate == null || $this->timePoint === null) {
```

Comment:
Is there a reason why for candidate `==` is used while `===` is used for timepoint? Would be better if both use `===`

Reply:
i agree. this seems to be a typo